target_redshift/__init__.py

```diff
@@ -340,15 +342,31 @@ def load_stream_batch(stream, records_to_load, row_count, db_sync, delete_rows=F
         row_count[stream] = 0
 
 
-def flush_records(stream, records_to_load, row_count, db_sync):
-    csv_fd, csv_file = mkstemp()
-    with open(csv_fd, 'w+b') as f:
+def flush_records(config, stream, records_to_load, row_count, db_sync):
```

Comment:
list of arguments changed by adding `config`arg but only here and not at the places where it's been called.